inference-engine/src/vpu/graph_transformer/src/blob_reader.cpp

```diff
@@ -142,8 +145,11 @@ void BlobReader::parse(const std::vector<char>& blob) {
         ie::TensorDesc ieDesc = DataDesc(dataType, dimsOrder, vpuDims).toTensorDesc();
         ie::Data outputData(outputName, ieDesc);
 
-        _networkOutputs[outputData.getName()]    = std::make_shared<ie::Data>(outputData);
+        if (outputData.getName().find("@shape") == std::string::npos) {
```

Comment:
Looks like a copy-paste from inputs. Should we remove code duplication?

Reply:
Agreed, done.

Reply:
I've fixed the previous version of this code to get rid of `bool isInput`, so now it looks better. Please take a look once more.